src/framework/piral-core/src/components/DefaultLayout.test.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { DefaultLayout } from './DefaultLayout';
+import { mount } from 'enzyme';
+
+describe('Default Layout Component', () => {
+    it('renders correctly for desktop', () => {
+      const children = <div></div>;
+      const node = mount(<DefaultLayout currentLayout="desktop" children={children}/>);
+      expect(node.find('div').length).toBe(1);
+    });
+
+    it('renders correctly for mobile', () => {
+        const children = <div></div>;
+        const node = mount(<DefaultLayout currentLayout="mobile" children={children}/>);
+        expect(node.find('div').length).toBe(1);
+    });
+
+    it('renders correctly for tablet', () => {
+      const children = <div></div>;
+      const node = mount(<DefaultLayout currentLayout="tablet" children={children}/>);
+      expect(node.find('div').length).toBe(1);
+    });
+
+    it('rendering reports error because of wrong element', () => {
+      const children = <div></div>;
+      const node = mount(<DefaultLayout currentLayout="tablet" children={<div></di>}/>);
```

Comment:
You meant `<div />` here? Because you wrote `<div></div>`

Reply:
Sorry, Typo, I just wanted to create an error case.

Reply:
But this is a compiler error!

This cannot be a case. Supply `null` or something like that if you want to have a real issue => your thing does not compile and thus does not emit code and thus cannot run at all.